vectordatasource/transform.py

```diff
@@ -1125,9 +1125,9 @@ def intracut(ctx):
 
 
 # place kinds, as used by OSM, mapped to their rough
-# scale_ranks so that we can provide a defaulted,
-# non-curated scale_rank / min_zoom value.
-_default_scalerank_for_place_kind = {
+# min_zoom so that we can provide a defaulted,
```

Comment:
I don't think this is necessary anymore as we explicitly call these out in the places.yaml file.

- https://github.com/tilezen/vector-datasource/blob/master/yaml/places.yaml#L48-L65